packages/graphql/src/types/output/Itinerary.js

```diff
@@ -11,6 +11,11 @@ export default new GraphQLObjectType({
   name: 'Itinerary',
   fields: {
     id: GlobalID(({ id }) => id),
+    price: { type: GraphQLInt },
```

Comment:
We will probably need to change this type for price as there would be currency added. @FilipMessa how do we want to deal with breaking changes in the GQL server? Do we want backwards compatibility? 
By using semver, we could always decide to do breaking changes and bump version?

Reply:
How about `@kiwicom/graphql-bc-checker`

Reply:
I would recommend you to not deal with breaking changes. It was in my opinion mistake to turn this GraphQL into the server because of issues like this and because of the fact you will have to deploy and keep it running (complexity of this approach is much higher). But honestly, you are your own consumers so please **do** breaking changes as needed. At least for now. 😊 

Eventually it would be great to use `@kiwicom/graphql-bc-checker` since it supports logging of breaking changes as well as detection of manual changes. No GitHub fancy integration though. At least not yet.

Reply:
Yeah, for me the problem with the inMemory fetcher was the lack of graphiql, but now I know we can have even that. 

Another concern was the bundle size, but I don't think we need to care about it  in this project.

Reply:
Not at this stage - make it work first. This is why it was like it was - just to iterate super fast without server needed (+ with the ability to very easily create a server from it). 🙃

Reply:
Yes, I understand your idea but we currently have the server and there is no reason to return back to the serverless state. 

> But honestly, you are your own consumers so please do breaking changes as needed. At least for now. 😊

this is good point  💯 , thanks @mrtnzlml